README.md

```diff
@@ -32,6 +32,10 @@ We appreciate your continued support, thank you!
 
 This library is broken up into several packages as a monorepo so that you only need to install the packages necessary for your use case. This README contains information pertaining to all packages. For examples on how to get started quickly, head over to the READMEs of each individual package (linked and described below), which includes detailed examples.
 
+## .env file
+
+You will find a file in the root of the project called .env_sample. You can run `mv .env_sample .env` or manually rename the file to `.env`; be sure to add your [API key](https://sendgrid.com/docs/User_Guide/Settings/api_keys.html#-Creating-an-API-key).
```

Comment:
It is better to copy the file and then edit it. This prevents subsequent updates to the dependency don't throw errors or get weird on someone's production server.

Can you please modify this from 
```
mv .env_sample .env
```
to 
```
cp .env_sample .env
```

Reply:
@mbernier thanks for the feedback, i've updated the readme.md with your request

Reply:
Awesome, thank you!